src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -1231,4 +1233,80 @@ private static List<Map<String, Object>> sortByValue(final List<Map<String, Obje
                 .sorted(Comparator.comparing(map -> (String) map.get(key)))
                 .collect(Collectors.toList());
     }
+
+    @Test
+    @ToggleEnable(mapKey = "OrganisationExternalController.deletePaymentAccountsOfOrganisation", withFeature = true)
+    @ExtendWith(FeatureToggleConditionExtension.class)
+    @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+    void deletePbaOfExistingOrganisationShouldBeForbiddenWhenLDOff() {
+        log.info("deletePbaOfExistingOrganisationShouldBeForbiddenWhenLDOff :: STARTED");
+
+        setUpTestData();
+        String bearerToken = inviteUser(puiFinanceManager, intActiveOrgId, generateRandomEmail(),
+                "firstName2", "lastName2");
+
+        PbaRequest deletePbaRequest = new PbaRequest();
+        deletePbaRequest.setPaymentAccounts(Set.of("PBA0000021", "PBA0000022", "PBA0000023"));
+
+        professionalApiClient.deletePaymentAccountsOfOrganisation(deletePbaRequest,
```

Comment:
this line making a call to existing external endpoint, so this test is invalid. make change to call new internal endpoint with prd-admin user